proxies/live/apiproxy/proxies/default.xml

```diff
@@ -20,41 +20,50 @@
       </Response>
       <Condition>request.verb == "OPTIONS" AND request.header.origin != null AND request.header.Access-Control-Request-Method != null</Condition>
     </Flow>
+    <Flow name="Flow.AuthApiKey">
+      <Request>
+        <Step>
+          <Name>VerifyApiKey-Header</Name>
+        </Step>
+      </Request>
+      <Response/>
+      <Condition>((proxy.pathsuffix MatchesPath "/_status")) and (request.verb = "GET")</Condition>
+    </Flow>
     <Flow name="Flow-GenerateAccessToken">
-            <Condition>(proxy.pathsuffix MatchesPath "/token") and (request.verb = "GET")</Condition>
```

Comment:
No for this PR but I think we need to sort out our indentation level for all of our xml files. They are all inconsistent.